src/Directory.Build.props

```diff
@@ -9,6 +9,8 @@
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <!-- To lock the version of Particular.Analyzers, for example, in a release branch, set this property in Custom.Build.props -->
     <ParticularAnalyzersVersion Condition="'$(ParticularAnalyzersVersion)' == ''">1.6.1</ParticularAnalyzersVersion>
+    <NServiceBusKey>0024000004800000940000000602000000240000525341310004000001000100dde965e6172e019ac82c2639ffe494dd2e7dd16347c34762a05732b492e110f2e4e2e1b5ef2d85c848ccfb671ee20a47c8d1376276708dc30a90ff1121b647ba3b7259a6bc383b2034938ef0e275b58b920375ac605076178123693c6c4f1331661a62eba28c249386855637780e3ff5f23a6d854700eaa6803ef48907513b92</NServiceBusKey>
+    <TestProjectKey>00240000048000009400000006020000002400005253413100040000010001007f16e21368ff041183fab592d9e8ed37e7be355e93323147a1d29983d6e591b04282e4da0c9e18bd901e112c0033925eb7d7872c2f1706655891c5c9d57297994f707d16ee9a8f40d978f064ee1ffc73c0db3f4712691b23bf596f75130f4ec978cf78757ec034625a5f27e6bb50c618931ea49f6f628fd74271c32959efb1c5</TestProjectKey>
```

Comment:
Nitpick, but why "project" in the name? I would instinctively named the key `Test.snk` and this property `TestKey`. If "project" is suitable here, that would seem to suggest that the other key should be named `NServiceBusProject.snk`.

FWIW, the current file in NServiceBus is named [`Test.snk`](https://github.com/Particular/NServiceBus/blob/master/src/Test.snk).

Reply:
> FWIW, the current file in NServiceBus is named Test.snk.

Yep, and the key in this PR is that key, but renamed.

We have quite a few Test.snk files already in repos, but as I discovered when I first had the idea to make the changes in this PR, they aren't all the same file. There are several different versions, with different keys.

This PR is an attempt to unify on a single test key, but just keeping the same name, Test.snk, would cause this PR to be breaking for all repos that aren't using this version of the key file already. It seemed to make more sense to avoid that and just choose a different name for the test key. That gets this key synced in all repos without disruption, and then a follow-up can switch to using the new key/method for specifying InternalsVisibleTo.

Reply:
Part of me thinks that having a list of red PR's to get through to make the switch would be better.

Reply:
FTR we discussed and agreed on `NServiceBusTestsKey` and `NServiceBusTests.snk`